### PR TITLE
feat: add support for namespaced imports

### PR DESCRIPTION
**Pull request by jocisneros**
Named imports can cause ridiculously huge import lines depending on the size of the graph. This change allows for the use of namespaced imports.

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -54,6 +54,26 @@ export interface ValidationSchemaPluginConfig extends TypeScriptPluginConfig {
...
+   *   path/to/schemas.ts:
+   *     plugins:
+   *       - graphql-codegen-validation-schema
+   *     config:
+   *       schema: yup
+   *       importFrom: ./path/to/types
+   *       namespacedImportName: types
+   * ```
+   */
+  schemaNamespacedImportName?: string
```
Comment (Code-Hex) [inline src/config.ts:76]: @jocisneros Could you add to also README?

**Review by Code-Hex — COMMENTED** [review]
LGTM if you fix my suggestion!

File: src/config.ts (hunk 1 above)
Comment (jocisneros) [reply src/config.ts:76]: just added 🫡

**Review by Code-Hex — APPROVED** [review]
Thank you!

**Pull request merged**